libkbfs/keybase_service_util.go

```diff
@@ -62,6 +62,19 @@ func serviceLoggedIn(ctx context.Context, config Config, name string,
 		}
 	}
 
+	if adminFeatureList[session.UID] {
+		log.CDebugf(ctx, "Enabling a dir op batch size of %d",
+			bgFlushDirOpBatchSizeDefault)
+		// NOTE: This overrides any command-line parameter.  It only
+		// matters until we un-feature-flag this, I think it's ok for
+		// now.
+		config.SetBGFlushDirOpBatchSize(bgFlushDirOpBatchSizeDefault)
```

Comment:
Does this mean all admins get the default batch size (which is 20) no matter the value of `-sync-batch-size`? It seems the command line flag doesn't do anything then does it?